lib/ruby-lint/analysis/loop_keywords.rb

```diff
@@ -57,7 +58,7 @@ class LoopKeywords < Base
       # @param [RubyLint::AST::Node] node
       #
       def verify_keyword(keyword, node)
-        if current_scope.type != :block and !@allow_keyword
+        if current_scope.type != :block && [0, nil].include?(@allow_keyword)
```

Comment:
Why is there a nil check here? I'd rather have `@allow_keyword` always set to a Fixnum so this can just be `and @allow_keyword == 0`. Also please keep the `and` instead of `&&`.